docs/app/documentation/component-docs/button-split/button-split-header/button-split-header.component.html

```diff
@@ -0,0 +1,11 @@
+<header>Button</header>
```

Comment:
Change to "Split Button", this also applies to every instance where "Button Split" is used. Fundamental-Styles calls it Split Button so we should follow their lead on this.